### ISSUE TITLE
[1.12][0.0.14.18] Crashing Upon World Load, Rendering Error?

### ISSUE THREAD

**Issue by Opulous** [Chisel-Team/Chisel#546]
Hello all, I'm a small private modpack maker and I've been having some crash issues with the latest 1.12 releases of Chisel. I normally don't like to bug devs about bugs since modpack debugging is an issue, but google searching this crash report actually turned up zero results, so I'm not sure where else to turn. I've tried this on all the 1.12 versions of chisel, and it's produced the same crash for each and every one. It crashes every single time I load a world, regardless of the world used.

Here's a pastebin of the crash report: https://pastebin.com/StyHi846

And here's the latest.log from forge: https://pastebin.com/eU0KEwii

I'm suspecting there's an issue with Chickenbones mods compatibility, based on excerpt from the latest.log:

> net.minecraft.util.ReportedException: Rendering item
>    at codechicken.lib.render.item.CCRenderItem.func_184391_a(CCRenderItem.java:227) 
~[CCRenderItem.class:?]

 and from the crash report:

> Stacktrace:
>    at codechicken.lib.render.item.CCRenderItem.func_184391_a(CCRenderItem.java:217)
from the crash report.

I'm not really a coder though, so this is just a guess. I'm currently running Forge1.12-14.21.1.2443 with chisel 0.0.14.18, CTM 0.2.2.7, and the latest versions of ChickenLib and Optifine.

**Comment by tterrag1098** [comment]
If you cannot reproduce this without CCL installed, I would assume the bug is on their end.

**Comment by Opulous** [comment]
I did some more testing this morning. First thing I did was try removing ChickenLib and ALL mods dependent on it, including EnderStorage, ChickenChunks, Draconic Evo, and all the CofH Thermal themed mods. Boom, game runs fine, no crashes. I made a world without the chicken-dependent mods in it and it runs great.

Next step, I tried putting all the chicken-dependent mods back into the modpack, but loading up the world that didn't have any of the CCL-dependant mods' worldgen features included. The world without those features actually still runs! So this seems to tell me that there's an item or a block specifically added to worldgen by CCL-dependent mods which fucks with Chisel's rendering. I confirmed this by once again trying to make a world with Chisel and the CCL mods present, and the crash returns. I'm going to try to narrow things down now by specifically testing individual CCL mods to see which one of them specifically is causing the issue.

**Comment by tterrag1098** [comment]
Well the crash is coming from one of the toasts that comes from unlocking a recipe, so it makes sense that it would only happen on a new world.

**Comment by Opulous** [comment]
Well that was a strange experience. I spent the last several hours trying to isolate which of the CCL-dependent mods was specifically throwing the crash, and it stopped happening entirely. No matter how many permutations I tried of the various mods, I can no longer reproduce the crash. It's super strange, but at the same time I'm not going to complain. I love all these mods and Chisel especially. I will close this issue since I can no longer reproduce it, thank you to tterrag for your help and time!
